src/android/toga_android/window.py

```diff
@@ -46,6 +46,10 @@ def set_content(self, widget):
     def set_title(self, title):
         pass
 
+    def get_position(self):
+        self.interface.factory.not_implemented('Window.get_position()')
```

Comment:
If you already implemented the method, no need for that warning :)
The same goes for the other platforms.